src/sentry/snuba/metrics/fields/snql.py

```diff
@@ -167,6 +167,19 @@ def all_transactions(org_id, metric_ids, alias=None):
     )
 
 
+def failure_count_transaction(org_id, metric_ids, alias=None):
+    return _dist_count_aggregation_on_tx_status_factory(
```

Comment:
I keep looking this up, so maybe a link to the definition of failure rate would be nice here

```suggestion
    # See https://docs.sentry.io/product/performance/metrics/#failure-rate
    return _dist_count_aggregation_on_tx_status_factory(
```

Reply:
Actually, this functionality is about the _count_, not the _rate_. But I take that suggestion and will add this comment in the failure PR.